### PR TITLE
added jenkins pub/private key vars to variables

### PR DESCRIPTION
**Pull request by cfoskin**
https://issues.jboss.org/browse/AGDIGGER-247

### CODE REVIEW

**Comment by cfoskin** [comment]
ping @aliok @PhilipGough

File: docs/digger/installation/installer-variables.asciidoc [hunk 1]
```diff
@@ -56,12 +56,21 @@ you must specify OAuth token so adding the key automatically doesn't fail
...
+
+|jenkins_private_key_path
+|Jenkins SSH private key path
+|~/.ssh/id_rsa
+|No
+
+|jenkins_public_key_path
+|Jenkins SSH public key path
+|~/.ssh/id_rsa.pub
+|No
```
Comment (PhilipGough) [inline docs/digger/installation/installer-variables.asciidoc]: Looks ok. Maybe a bit light on the description but for upstream docs I think this is fine. wdyt @aliok ?

Comment (cfoskin) [reply docs/digger/installation/installer-variables.asciidoc]: @PhilipGough I kept it similar to what was used in others but I will update it if @aliok and you agree I should

Comment (aliok) [reply docs/digger/installation/installer-variables.asciidoc]: Yep, absolutely. LGTM @cfoskin

**Pull request merged**